website/docs/mcu_kit/README.md

```diff
@@ -10,105 +10,147 @@ We sell a complete kit containing all necessary components on the Amiga shop web
 <p align="center">
 <img width="650" title="Male M12 CAN bus connector" alt="Screen Shot 2022-08-16 at 7 41 38 PM" src="https://user-images.githubusercontent.com/810997/185022824-593e543f-7899-4a65-93b0-9f07e97f8572.png"/>
 </p>
 
 
-### Flashing the UF2 firmware on the M4 device
-##### On initial plug in
+## Flashing the UF2 firmware on the M4 device
 
-Our goal is to have the Adafruit Feather show up as a CIRCUITPY directory for plugging it in, see 
-[this reference](https://learn.adafruit.com/circuit-playground-lesson-number-0/usb-connection).
+We run [CircuitPython](https://circuitpython.org/) on the microcontrollers,
+as it is well supported, well documented, and recommended by Adafruit for their microcontrollers.
 
-* Plug in your feather to your computer probably using a Type-C (feather) to Type-C (computer connector).
-   * You'll get randomly varying LED colors
-* Look for the little reset switch on the Feather board. Double click the reset button to enter bootloader mode 
-[(reference here)](https://learn.adafruit.com/circuit-playground-lesson-number-0/reset-button-bootloader)
-   * The Feather should automatically remount and show up as FTHRCANBOOT
-   * This allows you to copy a Microsoft standard flash format uf2 file onto the device
-* For each OS, see below how to copy (flash) the ``amiga-dev-kit/feather_m4_can/uf2s/adafruit-circuitpython-feather_m4_can-en_US-7.3.2.uf2`` file onto the Feather
-* After the above, feather should reboot with the newly loaded firmware and now show up as CIRCUITPY
+CircuitPython is installed by flashing the microcontroller with the correct UF2 file, and you can tell if CircuitPython is installed if the microcontroller is mounted as a directory called `CIRCUITPY` whenever you connect it.
 
-Below are instructions for flashing under different operating systems.
+### On first connection
 
+> The following instructions summarize the
+> [CircuitPython on Feather M4 CAN](https://learn.adafruit.com/adafruit-feather-m4-can-express/circuitpython-on-feather-m4-can)
+> instructions.
+>
+> You can also check out the more generic
+> [Adafruit Installing CircuitPython](https://learn.adafruit.com/welcome-to-circuitpython/installing-circuitpython) instructions.
 
-##### On subsequent plug in
-When attaching a previously flashed Feather M4 device, it should automatically mount as CIRCUITPY. 
+1. Connect your feather to your computer through the USB-C port on the feather
+> Check that the large LED is green or randomly varying colored. If it is red, you are most likely using a charge only USB-C cable.
 
-#### WSL uf2 flashing
+2. Double click the reset button to enter `BOOTLOADER` mode, allowing you to flash the microcontroller with a Microsoft standard flash format uf2 file. The Feather should automatically remount and show up as `FTHRCANBOOT` or `FEATHERBOOT`
+> *NOTE:* The timing of the double click can be a little tricky.
```

Comment:
https://docusaurus.io/docs/markdown-features/admonitions

We should check this (and others) because visually it makes a difference

Reply:
Oh those are nice. I'll go through and change them